sycl/source/detail/device_filter.cpp

```diff
@@ -91,7 +91,7 @@ device_filter::device_filter(const std::string &FilterString) {
       std::string Message =
           std::string("Invalid device filter: ") + FilterString +
           "\nPossible backend values are "
-          "{host,opencl,level_zero,cuda,hip,*}.\n"
+          "{host,opencl,level_zero,cuda,hip,esimd_emulator*}.\n"
```

Comment:
Late comment: This line would be better as below:
`"{host,opencl,level_zero,cuda,hip,esimd_emulator,*}.\n"`

Reply:
Will apply this change as piggyback in another PR later.